easybuild/easyconfigs/o/OpenCoarrays/OpenCoarrays-1.9.0-gompi-2017a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenCoarrays'
+version = '1.9.0'
+
+homepage = 'https://github.com/sourceryinstitute/opencoarrays'
+description = """OpenCoarrays is an open-source software project that supports
+the coarray Fortran (CAF) parallel programming features of the Fortran 2008
+standard and several features proposed for Fortran 2015 in the draft Technical
+Specification TS 18508 Additional Parallel Features in Fortran."""
+
+toolchain = {'name': 'gompi', 'version': '2017a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['%s/releases/download/%s' % (homepage, version)]
+
+sources = [SOURCE_TAR_GZ]
+
+separate_build_dir = True
+
+sanity_check_paths = {
+    'files': ['lib/%s' % x for x in ['libcaf_mpi.a']] +
+             ['include/%s' % x for x in ['opencoarrays.mod']] +
+             ['bin/%s' % x for x in ['caf', 'cafrun']],
```

Comment:
no need to use a list comprehension if there's only a single file in that subdir, so let's simplify (& sort alphabetically) to:

```python
    'files': ['bin/caf', 'bin/cafrun', 'include/opencoarrays.mod', 'lib/libcaf_mpi.a'],
```